### PR TITLE
Integration test fixes for Python 3.5

### PR DESCRIPTION
**Pull request by catleeball**

### CODE REVIEW

File: gslib/command_runner.py [hunk 1]
```diff
@@ -172,7 +172,8 @@ def __init__(self, bucket_storage_uri_class=BucketStorageUri,
     else:
       self.command_map = self._LoadCommandMap()
 
-  def _LoadCommandMap(self):
+  @staticmethod
```
Comment (houglum) [inline gslib/command_runner.py]: http://google.github.io/styleguide/pyguide.html#217-function-and-method-decorators

Why do we need the @staticmethod decorators? The style guide discourages them.

It looks like all usages of this function are via `self._LoadCommandMap()`, so we can just add self as a parameter.

File: gslib/tests/testcase/integration_testcase.py [hunk 2]
```diff
@@ -315,7 +315,8 @@ def ClearPOSIXMetadata(self, obj):
                     '-h', 'x-%s-meta-%s' % (provider_meta_string, MODE_ATTR),
                     suri(obj)])
 
-  def _ServiceAccountCredentialsPresent(self):
+  @staticmethod
+  def _ServiceAccountCredentialsPresent():
```
Comment (houglum) [inline gslib/tests/testcase/integration_testcase.py]: Same as other comment; add `self` to parameters.

File: gslib/tests/testcase/integration_testcase.py [hunk 3]
```diff
@@ -326,7 +327,8 @@ def _ServiceAccountCredentialsPresent(self):
     return (config.has_option('Credentials', 'gs_service_key_file') or
             config.has_option('GoogleCompute', 'service_account'))
 
-  def _ListBucket(self, bucket_uri):
+  @staticmethod
```
Comment (houglum) [inline gslib/tests/testcase/integration_testcase.py]: Same as other comment; add `self` to parameters.

File: gslib/tests/testcase/integration_testcase.py [hunk 4]
```diff
@@ -477,7 +479,8 @@ def VerifyRetentionPolicy(self,
       self.assertGreater(effective_time_in_seconds,
                          current_time_in_seconds - 60)
 
-  def DateTimeToSeconds(self, datetime_obj):
+  @staticmethod
+  def DateTimeToSeconds(datetime_obj):
```
Comment (houglum) [inline gslib/tests/testcase/integration_testcase.py]: Same as other comment; add `self` to parameters.

File: gslib/tests/testcase/integration_testcase.py [hunk 5]
```diff
@@ -1019,7 +1022,8 @@ def VerifyLocalPOSIXPermissions(self, path, gid=None, uid=None, mode=None):
     if mode is not None:
       self._VerifyLocalMode(path, mode)
 
-  def FlatListDir(self, directory):
+  @staticmethod
+  def FlatListDir(directory):
```
Comment (houglum) [inline gslib/tests/testcase/integration_testcase.py]: Same as other comment; add `self` to parameters.

File: gsutil.py [hunk 6]
```diff
@@ -75,25 +75,25 @@ def OutputAndExit(message):
...
-    ('boto', ''),
-    ('gcs-oauth2-boto-plugin', ''),
-    ('fasteners', ''), # oauth2client and apitools dependency
-    ('monotonic', ''), # fasteners dependency
-    ('httplib2', submodule_pyvers),
-    ('python-gflags', ''),
-    ('retry-decorator', ''),
-    ('six', ''),
-    ('socksipy-branch', ''),
+  ('argcomplete', ''),  # For tab-completion (gcloud installs only).
```
Comment (houglum) [inline gsutil.py]: This new indentation is wrong; we use 2-space indent for most cases, but we use 4 when continuing an expression (args to a function, adding items to a collection, etc.), we use 4 spaces.  This is especially helpful for differentiating multiline if statements from their bodies, e.g.:

```python
def Func():
  foo = 'bar'
  my_list = ['default_arg']
  # This is indented by 2 spaces
  if SomeConditionFunction(
      arg1,  # This was indented by 4 more spaces, starting from where the "if" was.
      arg2):
    foo = 'qux'  # Only indented 2 farther than the above "if".
    my_list = [
        # Continuation of the expression on the line above; this is indented by 4.
        'item1',
        'item2',
        'etc'
    ]
```

**Comment by catleeball** [comment]
I got added to the CLA group, so I'm going to close this PR and reopen it to see if it is any happier.

**Pull request closed without merging**